client/src/components/ManageAccount/tests/ManageAccountModals.test.js

```diff
@@ -52,6 +52,70 @@ const dummyState = {
 };
 
 const mockStore = configureStore({});
+describe ('testing multiple phone numbers', () =>  {
+  const testState = {
+    ...dummyState,
+    modals: {
+      optional: true
+    }
+  };
+  let getByLabelText;
+  beforeEach(() => {
+    const testStore = mockStore({
+      profile: testState
+    });
+    const utils = render(
+      <Provider store={testStore}>
+        <EditRequiredInformationModal />
+      </Provider>
+    );
+    getByLabelText = utils.getByLabelText;
+  });
+
+  const stateWithFields = {
+    ...testState,
+    fields: {
+      countries: [[230, 'United States']],
+      institutions: [[1, 'University of Texas at Austin']],
+      ethnicities: [['Decline', 'Decline to Identify']],
+      genders: [['Other', 'Other']],
+      professionalLevels: [['Other', 'Other']],
+      titles: [['Other User', 'Other User']]
+    }
+  };
+  const storeWithFields = mockStore({ profile: stateWithFields });
+  stateWithFields['data']['demographics']['phone'] = '321-234-2342'
+  var {queryByText} = render(
+    <Provider store={storeWithFields}>
+      <EditRequiredInformationModal />
+    </Provider>
+  );
+  const submitButton = getByLabelText(/required-submit/);
```

Comment:
@rstijerina @xandroc  I'm trying to figure out why I'm getting ` TypeError: getByLabelText is not a function`. Do you guys know what is causing the issue?

Reply:
Have you tried grabbing that function from the result of the render method above?
```
  var { getByLabelText, queryByText } = render(
    <Provider store={storeWithFields}>
      <EditRequiredInformationModal />
    </Provider>
  );
```

Reply:
@rstijerina I already tried that. Somehow I'm getting Unable to find a label with the text of: /required-submit/ for line 93

Reply:
the below fixes the search issue and works for testing multiple valid phone number and email formats by passing different params to the same bit of code

```
const validCases = [['223-043-3406', 'w@utexas.edu'],[ '1-541-754-3010', 'w@yahoo.com'],
  ['(541) 754-3010','w@gmail.com']];
describe('various valid input phone numbers and email addresses', () => {
    const testState = { ...dummyState, modals: { required: true } };
  test.each(validCases)(
    "given number and and emails should be valid",
    async (phoneNumber, email) => {
    const stateWithFields = {
      ...testState,
      fields: {
        countries: [[230, 'United States']],
        institutions: [[1, 'University of Texas at Austin']],
        ethnicities: [['Decline', 'Decline to Identify']],
        genders: [['Other', 'Other']],
        professionalLevels: [['Other', 'Other']],
        titles: [['Other User', 'Other User']],
      }
    };
    const storeWithFields = mockStore({ profile: stateWithFields });
    stateWithFields['data']['demographics']['phone'] = (phoneNumber);
    stateWithFields['data']['demographics']['email'] = (email);
    const {getByLabelText ,queryByText} = render(
      <Provider store={storeWithFields}>
        <EditRequiredInformationModal />
      </Provider>
    );
    const submitButton = getByLabelText(/required-submit/);
    const clickSpy = jest.spyOn(submitButton, 'click');
    fireEvent.click(submitButton);
    await wait(() => {
      expect(queryByText('Phone number is not valid')).toBeNull();
      expect(queryByText('Please enter a valid email address')).toBeNull();
      expect(clickSpy)
    });
    }
  );
});
```